CrowdStrike/ember-toucan-core
# chore: add `@noResultsText` argument to Multiselect

## 🚀 Description

We want to increase consistency where we can. In the case of what's displayed when there are no results, we (@clintcs, @ynotdraw)—after some discussion—_think_ we can get away with being prescriptive and forgo a named block (`:noResults`) in favor of an argument (`@noResultsText`). This will also bring Multiselect in line with Autocomplete, which also supports `@noResultsText`.


## 🔬 How to Test

1. Navigate to the [preview URL](https://8a5dc82c.ember-toucan-core.pages.dev/docs/components/multiselect).
1. Filter a demo until there are no results.
1. Verify that "No results" is shown in the popover.
